deploy/operator/setup_assisted_operator.sh

```diff
@@ -0,0 +1,134 @@
+source utils.sh
+
+set -xeo pipefail
+
+SERVICE_IMAGE="${SERVICE_IMAGE:-}"
+INSTALLER_IMAGE="${INSTALLER_IMAGE:-}"
+AGENT_IMAGE="${AGENT_IMAGE:-}"
+DATABASE_IMAGE="${DATABASE_IMAGE:-}"
+CONTROLLER_IMAGE="${CONTROLLER_IMAGE:-}"
+OPENSHIFT_VERSIONS="${OPENSHIFT_VERSIONS:-}"
+
+ASSISTED_NAMESPACE="${ASSISTED_NAMESPACE:-assisted-installer}"
+OPERATOR_INDEX="${OPERATOR_INDEX:-quay.io/ocpmetal/assisted-service-index:latest}"
+
+function subscription_config() {
+    if [ -n "${SERVICE_IMAGE}" ]; then
+cat <<EOF
+    - name: SERVICE_IMAGE
+      value: '$SERVICE_IMAGE'
+EOF
+    fi
+
+    if [ -n "${INSTALLER_IMAGE}" ]; then
+cat <<EOF
+    - name: INSTALLER_IMAGE
+      value: '$INSTALLER_IMAGE'
+EOF
+    fi
+
+    if [ -n "${AGENT_IMAGE}" ]; then
+cat <<EOF
+    - name: AGENT_IMAGE
+      value: '$AGENT_IMAGE'
+EOF
+    fi
+
+    if [ -n "${DATABASE_IMAGE}" ]; then
+cat <<EOF
+    - name: DATABASE_IMAGE
+      value: '$DATABASE_IMAGE'
+EOF
+    fi
+
+    if [ -n "${CONTROLLER_IMAGE}" ]; then
+cat <<EOF
+    - name: CONTROLLER_IMAGE
+      value: '$CONTROLLER_IMAGE'
+EOF
+    fi
+
+    if [ -n "${OPENSHIFT_VERSIONS}" ]; then
+cat <<EOF
+    - name: OPENSHIFT_VERSIONS
+      value: '$OPENSHIFT_VERSIONS'
+EOF
+    fi
+}
+
+echo "Applying CatalogSource for assisted-service-operator bundle..."
```

Comment:
I think those `echos` are unnecessary since `oc apply` is going to print what happened.